proto/reaction.proto

```diff
@@ -212,9 +233,9 @@ message CompoundPreparation {
     REPURIFIED = 3;  // Compound repurified (e.g., recrystallized).
     SPARGED = 4;  // Compound sparged, most likely to be the case with solvents.
     DRIED = 5;  // Moisture removed, e.g., using molecular sieves.
-    SYNTHESIZED = 6;  // Compound synthesized in-house
+    SYNTHESIZED = 6;  // Compound synthesized in-house.
   }
-  PreparationType type = 1;
+  repeated PreparationType type = 1;
```

Comment:
Rather than making the `PreparationType type` repeated, let's make `CompoundPreparation preparation = 7;` in the `Compound` message repeated (and rename to `preparations`). That way, each preparation type will get its own `details` field and a `CompoundPreparation` message is just a single preparative step.

Reply:
Agreed, sounds good.